spec/factories/users.rb

```diff
@@ -1,5 +1,5 @@
 FactoryGirl.define do
-  factory :user do
+  factory :user, aliases: [:agent] do
```

Comment:
Allows to refer to the factory as `agent`. e.g. in sponsors factory and in the sponsor feature.